app/src/test/java/com/eis/geoCalendar/gps/GPSPositionTest.java

```diff
@@ -0,0 +1,77 @@
+package com.eis.geoCalendar.gps;
+
+import android.location.Location;
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.powermock.api.easymock.PowerMock;
+import org.powermock.api.mockito.PowerMockito;
+import org.powermock.core.classloader.annotations.PrepareForTest;
+import org.powermock.modules.junit4.PowerMockRunner;
+
+import static org.easymock.EasyMock.expect;
+import static org.mockito.Mockito.doNothing;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.spy;
+import static org.mockito.Mockito.when;
+
+/**
+ * GPSPosition tests
+ *
+ * @author Alberto Ursino
+ * @since 12/28/2019
+ */
+@RunWith(PowerMockRunner.class)
+@PrepareForTest(Location.class)
+public class GPSPositionTest {
+
+    private GPSPosition gpsPosition1;
+    Location location1 = PowerMockito.mock(Location.class);
+    private final double DEF_LATITUDE = 100;
+    private final double DEF_LATITUDE_2 = 101;
+    private final double DEF_LONGITUDE = 100;
+    private final double DEF_LONGITUDE_2 = 101;
+    private final double DELTA = 0;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [This final field could be made static](https://app.codacy.com/manual/CremaLuca/geoCalendar/pullRequest?prid=4964556)